src/adapter/src/coord/sequencer.rs

```diff
@@ -1799,6 +1799,8 @@ impl<S: Append + 'static> Coordinator<S> {
         self.catalog_transact(Some(session), ops, |_| Ok(()))
             .await?;
 
+        fail::fail_point!("after_catalog_drop_replica");
+
         for (compute_id, replica_id) in replicas_to_drop {
             self.drop_replica(compute_id, replica_id).await.unwrap();
         }
```

Comment:
Would it make sense to have a 3rd failpoint after the `self.drop_replica` loop and then a 4th invocation of the test?

Reply:
I add a test run for this. I think it will not expose new scenarios, but better be safe (unless we are constrained on test duration time)